packages/deno/src/generators/application/schema.json

```diff
@@ -42,6 +42,15 @@
       "type": "boolean",
       "description": "Default the project to watch for file changes for the 'serve' target. For projects that do not need watch capability (eg: CLI projects), use --with-watch=false",
       "default": true
+    },
+    "monorepo": {
+      "type": "boolean",
+      "description": "Creates an integrated monorepo.",
+      "aliases": ["integrated"]
+    },
+    "rootProject": {
+      "type": "boolean",
+      "x-priority": "internal"
```

Comment:
question: what is the expected usage of these flags in terms of making a new app.

is the behavior always to have a integrated setup unless they a standalone project was made via the preset? 

the --monorepo flag doesn't make much sense to me as to make an app you already have to have a repo setup. 

and --rootProject really only matters for the preset as non one else should do it. couldn't we just use the face that the projects root dir is the root '.' instead of passing a flag?

Reply:
It's only for running with `create-nx-workspace`, and it won't really work for subsequent apps. It's the same way we did the other app generators like React.